screeps-game-api/src/lib.rs

```diff
@@ -11,13 +11,71 @@ extern crate serde_derive;
 #[macro_use]
 extern crate stdweb;
 
+/*  Used to get data from a javascript reference back into rust code. 
+
+Macro syntax (`$name` are expressions): 
+js_unwrap!($jsExpr)
+
+For reference, `js!()` is a macro that returns a `stdweb::Value` enum.
+https://docs.rs/stdweb/0.4.8/stdweb/enum.Value.html
+
+Here, `js_unwrap!()` takes any valid javascript expression (expresses a value) 
+and will attempt conversion to the receiving variable type using `try_into`. 
+For example:
+```
+let s: u32 = js_unwrap!(Game.time);
+```
+
+This will be be converted to 
+```
+let s: u32 = js!(Game.time).try_into().expect('Some Err Msg');
```

Comment:
I think this should be `js!(return Game.time)` rather than `js!(Game.time)`. The latter will not actually work, since `return` is necessary to return any values.

That was part of the motivation for having `js_unwrap!` - not having write `return ` everywhere.